data/interactive-defaults.ks

```diff
@@ -5,7 +5,7 @@ firstboot --enable
 
 %anaconda
 # Default password policies
-pwpolicy root --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy user --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy luks --strict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy root --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy user --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy luks --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
```

Comment:
I'd rather see the last one (LUKS) untouched, but if that's FESCO's decision, it's FESCO's decision. Just note that LUKS passphrases didn't have the "double click policy" in previous Fedora releases at all.